proxy/src/coco/source.rs

```diff
@@ -313,7 +333,7 @@ pub fn local_state(repo_path: &str) -> Result<LocalState, error::Error> {
     let repo = git::Repository::new(repo_path)?;
     let browser = Browser::new(&repo, "master")?;
```

Comment:
TODO(finto): This should be the default branch of the project, possibly.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/636/commits/b2536a4d58f78b8368eb9c521e5efe8932eb2787